cmd/logs.go

```diff
@@ -1,21 +1,49 @@
 package cmd
 
 import (
-	"fmt"
-
-	"github.com/spf13/cobra"
+		"fmt"
+		"encoding/json"
+		"github.com/spf13/cobra"
 )
 
 // logsCmd represents the logs command
 var logsCmd = &cobra.Command{
-	Use:   "logs",
-	Short: "View output from containers",
-	Long:  ``,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("TODO")
-	},
+		Use:   "logs",
+		Short: "View output from containers",
+		Long:  ``,
+		Run: logs,
 }
 
 func init() {
-	RootCmd.AddCommand(logsCmd)
+		RootCmd.AddCommand(logsCmd)
+}
+
+func logs(cmd *cobra.Command, args []string) {
+		//read the harbor compose file
+		var harborCompose = DeserializeHarborCompose(HarborComposeFile)
+		//iterate shipments
+		for shipmentName, shipment := range harborCompose.Shipments {
+			fmt.Println("Logs For:  " + shipmentName + " " + shipment.Env)
+			helmitObject := HelmitResponse{}
+			var response = GetLogs(shipment.Barge, shipmentName, shipment.Env)
+			err := json.Unmarshal([]byte(response), &helmitObject)
+			if err != nil {
+			    fmt.Println(err)
+			}
+			for _, provider := range helmitObject.Replicas {
+					fmt.Println("--------------------------------------------------------------------------------------------------------------")
```

Comment:
Might be nicer if these lines were a little shorter

```
----------
--- Host:  aaa.bbb.ccc.ddd
----------
--- Name:  foo-bar-app
--- Id:    asdfghjkl
--- Image: lorum-ipsum
----------
```

As is, a small terminal window will really bork the display